cmd/run.go

```diff
@@ -344,6 +356,12 @@ func runServer(ctx context.Context, config *Config) error {
 		opts := []mysql.MySQLOption{
 			mysql.WithLogger(logger),
 			mysql.WithTracer(tracer),
+			mysql.WithMaxTuplesPerWrite(config.MaxTuplesPerWrite),
```

Comment:
We must always pass these options so does it make more sense to just add them in the config?

Reply:
Ah, right. That makes sense. Thanks for the explanation.